lib/matplotlib/tight_bbox.py

```diff
@@ -1,5 +1,5 @@
 """
-This module is to support the *bbox_inches* parameter in `.Figure.savefig`.
+Support module for the *bbox_inches* parameter in `.Figure.savefig`.
```

Comment:
```suggestion
Helper module for the *bbox_inches* parameter in `.Figure.savefig`.
```
I have the impression that "helper module" is more common than "support module", but not sure.